src/config/locales/en-US.json

```diff
@@ -360,6 +360,7 @@
     "remove_alert": "Are you sure you want to remove?",
     "clear_alert": "Are you sure you want to clear?",
     "clear_user_alert": "Are you sure you want to clear all user data?",
+    "decrypt_fail_alert":"Bad previous app data, you will have relogin to reset app data",
```

Comment:
let's change string to this: `Corrupt app state, please relogin to reset state.`